### PR TITLE
fix: Close fetch response on HTTP broadcast sent

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Close fetch response on HTTP broadcast sent

### CODE REVIEW

File: src/RealtimeChannel.ts [hunk 1]
```diff
@@ -464,8 +464,10 @@ export default class RealtimeChannel {
         )
 
         if (response.ok) {
+          await response.body?.cancel()
           return 'ok'
         } else {
+          await response.body?.cancel()
           return 'error'
```
Comment (abc3) [inline src/RealtimeChannel.ts:471]:
```suggestion
await response.body?.cancel()
return response.ok ? 'ok' : 'error'
```

Comment (abc3) [inline src/RealtimeChannel.ts:471]:
```suggestion
await response.body?.cancel()
return response.ok ? 'ok' : 'error'
```

**Review by abc3 — APPROVED** [review]

File: src/RealtimeChannel.ts (hunk 1 above)
Comment (filipecabaco) [reply src/RealtimeChannel.ts:471]: way cleaner!

File: src/RealtimeChannel.ts [hunk 2]
```diff
@@ -464,9 +464,8 @@ export default class RealtimeChannel {
         )
 
         if (response.ok) {
-          return 'ok'
-        } else {
-          return 'error'
```
Comment (abc3) [inline src/RealtimeChannel.ts:469]: indentation 😬

File: src/RealtimeChannel.ts [hunk 3]
```diff
@@ -464,9 +464,8 @@ export default class RealtimeChannel {
...
+await response.body?.cancel()
+return response.ok ? 'ok' : 'error'
         }
       } catch (error: any) {
```
Comment (abc3) [inline src/RealtimeChannel.ts:470]: indentation 😬

Comment (abc3) [reply src/RealtimeChannel.ts:470]: and need remove `if`

**Pull request merged**